### PR TITLE
Actions workflow - update checkout version

### PR DESCRIPTION
**Pull request by ScottBrenner**
### First time contributor checklist:

- [x] I have read the [README](https://github.com/digidem/mapeo-desktop/blob/master/README.md) 

### Contributor checklist:

- [x] My contribution is **not** related to translations. _Please submit translation changes via our [Mapeo Crowdin project](https://crowdin.com/project/mapeo-desktop)_
- [x] My commits are in nice logical chunks with [good commit messages](http://chris.beams.io/posts/git-commit/)
- [ ] I have walked through the [QA Manual Testing
  Script](https://github.com/digidem/mapeo-desktop/blob/master/docs/testing.md) and updated it if necessary.
- [ ] If my changes depend upon an update to a dependency, I have updated the package-lock.json file using `npm install --package-lock`
- [ ] My changes have been tested with the [mobile app](https://github.com/digidem/mapeo-mobile/releases). 
- [ ] My changes are ready to be shipped to users on Windows, Mac, and Linux

### Description

Proposing updating the version of the `checkout` Action here to the latest - `v2` (https://github.com/actions/checkout#checkout-v2)/

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
         env:
           GITHUB_CONTEXT: ${{ toJson(github) }}
         run: echo "$GITHUB_CONTEXT"
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
         with:
           fetch-depth: 1
```
Comment (gmaclennan) [inline .github/workflows/nodejs.yml]: Thanks @ScottBrenner! This fetch-depth is no longer needed for v2 (it's the default) so you can remove this too.

Comment (ScottBrenner) [reply .github/workflows/nodejs.yml]: On it! https://github.com/digidem/mapeo-desktop/pull/285/commits/270b691533e57eb7223ebcdddaa8a589c1133164

**Comment by okdistribute** [comment]
This seems to not be finishing now, I wonder what happened?

**Comment by ScottBrenner** [comment]
@okdistribute Something broken here? 😐

**Comment by okdistribute** [comment]
hmm it seems like the branch is unknown somehow! can you make a new fork or branch and then push it again?

**Comment by ScottBrenner** [comment]
Right, deleted my fork - closing in favor of https://github.com/digidem/mapeo-desktop/pull/293

**Pull request closed without merging**